en/6-use-open-standards-solutions.md

```diff
@@ -152,13 +134,12 @@ In order to limit costs, avoid duplication of effort and provide a consistent cl
 
 **[TODO: Add/revise checklist items]**
 
+- follow government better practice and standards in the design of the service **(Digital Service Standard (AU))**
+- identify tools, systems, processes that can be adopted or reused from other services **(Digital Service Standard (AU))**
+- search for similar solutions in other jurisdictions. **(Digital Service Standard (AU))**
+- addressing any common user needs in a way that is consistent with the rest of government. **(Digital Service Standard (AU))**
 - Choose software frameworks that are commonly used by private-sector companies creating similar services **(Digital Services Playbook (US))**
-- Whenever possible, ensure that software can be deployed on a variety of commodity hardware types **(Digital Services Playbook (US))**
 - Ensure that each project has clear, understandable instructions for setting up a local development environment, and that team members can be quickly added or removed from projects **(Digital Services Playbook (US))**
-- [Consider open source software solutions](https://www.obamawhitehouse.gov/sites/default/files/omb/assets/egov_docs/memotociostechnologyneutrality.pdf) at every layer of the stack **(Digital Services Playbook (US))**
-- {: .dpgn-onegc-architectural } Build standards-based solution **(Build It Right - OneGC Architectural Checklist (draft))**

```

Comment:
Since this was moved to Guideline 6.1, you should move the related item in the Similar resources section to 6.1 as well.

Reply:
Sorry, I just noticed that I wasn't very clear with my previous comment. Please move the "{: .dpgn-onegc-architectural } Build standards-based solution..." item in the Similar resources sections from 6.2 to 6.1 since the related checklist item was moved.

Reply:
I think I got it